iOS/Flows/Threads/MessageTextView.swift

```diff
@@ -25,54 +25,7 @@ class MessageTextView: TextView {
     }
 
     func setText(with message: Messageable) {
-        switch message.kind {
-        case .text(_):
-            self.setText(message.kind.text)
-        case .attributedText(_):
-            break
-        case .photo(photo: let photo, body: let body):
-            guard let url = photo.url else { return }
-            self.loadImage(with: url, with: body)
-        case .video(video: let video, body: let body):
-            break
-        case .location(_):
-            break
-        case .emoji(_):
-            break
-        case .audio(_):
-            break
-        case .contact(_):
-            break
-        case .link(_):
-            break
-        }
-        
-    }
-    
-    private func loadImage(with url: URL, with body: String) {
```

Comment:
As you're aware, I decided it wasn't worth trying to put the images in the text view.